### ISSUE TITLE
is there a bug in pwndbg/pwndbg/commands/heap.py?

### ISSUE THREAD

**Issue by init13** [pwndbg/pwndbg#560]
when i use 'fastbins' it tell me TypeError: 'int' object is not subscriptable,so i checked the heap.py. if i change chain_fd[0] into chain_fd it can correct show me the fastbins, but will it chain_fd is not INT type in some case ? 
![heap](https://user-images.githubusercontent.com/35521617/47601767-e10aae80-da07-11e8-94bc-aca597ab81ca.png)
![heap2](https://user-images.githubusercontent.com/35521617/47601788-31820c00-da08-11e8-82c0-1b9f384cd2f2.png)

**Comment by disconnect3d** [comment]
Can you provide a minimal working example?

cc: @GrosQuildu can you look over it? :p

**Comment by init13** [comment]
```
int main ()
{
        char *p = (char *) malloc (20);
        free (p);
        return 0;
}
```
btw i installed peda and libheap before, but i commented it in .gdbinit.

**Comment by init13** [comment]
well....., i found another problem in here,this time i just want execute the example in "https://github.com/shellphish/how2heap/blob/master/glibc_2.25/fastbin_dup_into_stack.c" i set the break point in line 38.i check the heap when program interrupt at line 38,it tell me "ValueError: too many values to unpack (expected 3)"
![bins](https://user-images.githubusercontent.com/35521617/47615419-7b8eee80-dae9-11e8-8765-340b82f18206.png)
P.S i am a green hand in linux heap , i just want have a good environment to learn it.i am too tired to find GDB plug-in. my glibc version is (Ubuntu GLIBC 2.23-0ubuntu10) 2.23,and python version is 3.5.

**Comment by disconnect3d** [comment]
-Also are you using the latest dev branch?-
EDIT: Nvm, see below.

**Comment by GrosQuildu** [comment]
My fault, fix in #563

**Comment by disconnect3d** [comment]
@init13 shall be fixed by #563, can u check it out?

**Comment by init13** [comment]
yes,thank you very much!!
